wouso/resources/templates/cpanel/races/all.html

```diff
@@ -22,45 +22,191 @@
 {% endblock %}
 
 {% block sectioncontent %}
-<ul>
+<div class="right">
+    <button class="btn btn-sm btn-default btn-toggle" id="toggle_all_groups" data-show="1">
```

Comment:
@catalin-oancea data-show="1" can be confusing. Use true instead, or add the hidden class, and then check for it instead.

if(element.hasClass(hidden)) {
  // do something here
}
else {
  // do something else
}